src/chakra-components/menu.tsx

```diff
@@ -438,8 +448,9 @@ export const Option = <
       )}
       sx={sx}
       ref={innerRef}
-      data-disabled={isDisabled ? true : undefined}
```

Comment:
I realized there was no reason to use both `data-disabled` and `aria-disabled`. The `data-` props Chakra looks at for styling seem to be more of a fallback if the `aria-` props aren't applicable to be applied to a component.